src/styles/css/index.css

```diff
@@ -45,3 +52,49 @@ code {
 .grey-bg {
   background: #f5f6fa;
 }
+
+.btn-primary {
+  color: #fff;
+  background-color: #007742;
+  border: 2px solid #007742;
+  border-radius: 5px;
+}
+
+.btn-primary:hover {
+  background-color: #007742;
+  border: 2px solid #007742;
+  box-shadow: 0 5px 14px 0 #67a7876b;
+}
+
+/* TABLES */
+
+.ReactTable {
+  padding: 24px;
+  background-color: #ffffff;
+  border-radius: 5px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.ReactTable .rt-tbody .rt-td {
+  border-right: 1px solid rgba(0, 0, 0, 0.02);
+  padding: 8px 0px;
+}
+
+span.status-badge.active {
```

Comment:
Maybe make this more specific with `.ReactTable  span.status-badge.active`? Same with the next rule